tools/datagen/src/bin/datagen.rs

```diff
@@ -207,7 +207,11 @@ fn main() -> anyhow::Result<()> {
         .get_matches();
 
     match matches.occurrences_of("VERBOSE") {
-        0 => SimpleLogger::from_env().init().unwrap(),
+        0 => SimpleLogger::new()
+            .env()
+            .with_level(log::LevelFilter::Info)
+            .init()
+            .unwrap(),
         1 => SimpleLogger::new()
```

Comment:
Nit: Now that Info is the default, consider promoting Trace to a single v and drop the double vv